helper.go

```diff
@@ -0,0 +1,21 @@
+package weave
+
+import abci "github.com/tendermint/tendermint/abci/types"
+
+// CommitInfoFromABCI converts abci commit info to weave native type.
+// This struct represents validator signatures on the current block.
+func CommitInfoFromABCI(info abci.LastCommitInfo) CommitInfo {
```

Comment:
Yeah, I am happy to have this a native weave type, even if we mirror their types.
Just to keep our API clean.

In the end, even if we do something like

`type CommitInfo = abci.CommitInfo`

and everyone imports weave.CommitInfo, then at least we decouple it there, so we could redefine CommitInfo in weave and then all packages use the new definition (complete with compile errors if there are inconsistent changes)